src/Core/Grid/Query/AuthorizedApplicationQueryBuilder.php

```diff
@@ -108,10 +108,9 @@ private function applyAssociatedQueries(QueryBuilder $queryBuilder): void
     private function appendActiveApiAccessQuery(QueryBuilder $queryBuilder): void
     {
         $shopQueryBuilder = $this->getQueryBuilder()
-            ->select('count(api.active)')
+            ->select('count(api.enabled)')
```

Comment:
```suggestion
            ->select('count(api.id_api_access)')
```
Seems like it makes more sense to test the count of IDs no? Like in the other method